server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java

```diff
@@ -421,6 +434,54 @@ public void getQuotaShouldReturnBothWhenValueSpecified() throws Exception {
         softly.assertThat(jsonPath.getLong("global." + COUNT)).isEqualTo(22);
     }
 
+    @Test
+    public void getQuotaShouldReturnOccupation() throws Exception {
+        maxQuotaManager.setMaxStorage(userQuotaRootResolver.forUser(BOB), QuotaSize.size(80));
+        maxQuotaManager.setMaxMessage(userQuotaRootResolver.forUser(BOB), QuotaCount.count(100));
+        Mockito.when(currentQuotaManager.getCurrentStorage(any())).thenReturn(QuotaSize.size(40));
+        Mockito.when(currentQuotaManager.getCurrentMessageCount(any())).thenReturn(QuotaCount.count(20));
```

Comment:
here you override what's set it setup?

Reply:
Yes.  It's set up to return nothing. And in these tests I want something...

Reply:
ok in these cases I thought we did not define the value in the setup to be clear about the intent